src/it/scala/com/exasol/cloudetl/kafka/docker/BaseDockerIntegrationTest.scala

```diff
@@ -0,0 +1,125 @@
+package com.exasol.cloudetl.kafka
+
+import java.io.File
+import java.nio.file.Paths
+
+import com.exasol.containers.ExasolContainer
+import com.exasol.dbbuilder.dialects.Column
+import com.exasol.dbbuilder.dialects.exasol.ExasolObjectFactory
+import com.exasol.dbbuilder.dialects.exasol.ExasolSchema
+import com.exasol.dbbuilder.dialects.exasol.udf.UdfScript
+
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.funsuite.AnyFunSuite
+
+trait BaseDockerIntegrationTest extends AnyFunSuite with BeforeAndAfterAll {
+  private[this] val JAR_DIRECTORY_PATTERN = "scala-"
+  private[this] val JAR_NAME_PATTERN = "exasol-kafka-connector-extension-"
+  private[this] val DEFAULT_EXASOL_DOCKER_IMAGE = "7.0.10"
+
+  val network = DockerNamedNetwork("kafka-it-tests", true)
+  val exasolContainer = {
+    val c: ExasolContainer[_] = new ExasolContainer(getExasolDockerImageVersion())
+    c.withExposedPorts(8563, 2580)
+    c.withNetwork(network)
+    c.withReuse(true)
+    c
+  }
+  var factory: ExasolObjectFactory = _
+  var schema: ExasolSchema = _
+  val assembledJarName = getAssembledJarName()
+
+  override def beforeAll(): Unit =
+    exasolContainer.start()
+
+  override def afterAll(): Unit =
+    exasolContainer.stop()
+
+  def prepareExasolDatabase(schemaName: String): Unit = {
```

Comment:
Maybe rename to "installKafkaConnector" or something similar?

Reply:
Done.